pkgs/development/python-modules/beancount-parser/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, isPy3k
+, lark
+, poetry
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "beancount-parser";
+  version = "0.1.19";
+
+  disabled = !isPy3k;
```

Only [Python 3.9 and above](https://github.com/LaunchPlatform/beancount-parser/blob/master/pyproject.toml#L11) is supported.

```suggestion
  disabled = pythonOlder "3.9";
```